components/auto-complete/AutoComplete.tsx

```
@@ -16,6 +16,7 @@ import type {
   RefSelectProps,
   SelectProps,
   SelectSemanticAllType,
+  SearchConfig
```

ESLint

[error] Expected "SearchConfig" to come before "SelectSemanticAllType".

(perfectionist/sort-named-imports)